feg/radius/src/run.sh

```diff
@@ -13,13 +13,6 @@
 ################################################################################
 
 OUTPUT_DIR=bin
```

Comment:
Just tested, the output us placed under `radius` and not `bin`:
```suggestion
OUTPUT_DIR=radius
```